Lib/posixpath.py

```diff
@@ -552,8 +552,10 @@ def commonpath(paths):
                 common = s1[:i]
                 break
 
-        prefix = sep if isabs else sep[:0]
         return prefix + sep.join(common)
     except (TypeError, AttributeError):
-        genericpath._check_arg_types('commonpath', *paths)
+        genericpath._check_arg_types('commonpath', *(
+            # Can't use paths, can be an iterable
```

Comment:
Why not?

Reply:
I forgot to change this when I convert overwrote `paths` with the tails. It can simply be passed to the function without further changes.